vtex/faststore
# Feat: Adds `Slider` & `PriceRange` component

## What's the purpose of this pull request?

- [x] Moves `Slider` component to @faststore/components
- [x] Extracts `Slider` styles from starter to @faststore/ui
- [x] Adds `Slider` doc page
- [x] Refactor `PriceRange` component and add to @faststore/components
- [x] Adds  `PriceRange` styles to @faststore/ui
- [x] Adds `PriceRange` doc page

How to test it?

run `yarn workspace site dev` and check the Slider and PriceRange components  doc page or test it using the doc preview link: 
- [Slider Component](https://faststore-site-git-feat-slider-component-fs-764-faststore.vercel.app/components/atoms/slider)
- [PriceRange Component](https://faststore-site-git-feat-slider-component-fs-764-faststore.vercel.app/components/organisms/price-range)

### Starters Deploy Preview

[Core Replaces PriceRange FSUI ](https://github.com/vtex-sites/nextjs.store/pull/348)

<img width="32" alt="slider_controler_left" src="https://user-images.githubusercontent.com/3356699/211866912-a53e5a66-506e-4752-bbc3-b50089fd8fce.png">
